### PR TITLE
Fix functional composition methods

### PR DESCRIPTION
**Pull request by kosciak9**
Currently `flow` method done and `hex2string` skeleton method done. Still working on `compose` and `link`.

- [x] Running `yarn lint` does not trigger any linter errors
- [x] The test of the method you have fixed is passing
- [x] You have written a new skeleton method for someone else to work on!
- [x] You have written tests to accompany your skeleton method

#155

### CODE REVIEW

File: test/hex2string.test.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const { hex2string } = require("../lib");
+
+
+describe("flow", () => {
```
Comment (ktilcu) [inline test/hex2string.test.js]: This should be `hex2string`.

File: test/hex2string.test.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  it("should be a function", () => {
+    expect(hex2string()).toBeInstanceOf(Function);
+    expect(hex2string("001100")).toBeInstanceOf(Function);
```
Comment (ktilcu) [inline test/hex2string.test.js]: I think this should be an instance of string.

File: test/hex2string.test.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  });
+
+  it("when no string is provided, or string is empty,"
+     + " returns empty string", () => {
+    expect(hex2string("")).toBe("");
+    expect(hex2string()).toBe("");
+  });
+
+  it("should convert from hexadecimal to ascii", () => {
```
Comment (ktilcu) [inline test/hex2string.test.js]: if we are going hex => ascii maybe that should be the name of the function?

**Comment by BlakeGuilloud** [comment]
Hey @kosciak9 ! Could you please make the requested changes so we can get this func merged in? Thanks!!

**Comment by ktilcu** [comment]
closing this PR due to inactivity

**Pull request closed without merging**